client/package.json

```diff
@@ -5,6 +5,7 @@
   "prettier": "@blockstack/prettier-config",
   "dependencies": {
     "@fortawesome/free-brands-svg-icons": "^5.12.1",
+    "@material-ui/core": "^4.9.7",
```

Comment:
Just curious, does using Material-UI introduce a second UI framework along with React Bootstrap? Or is this just a library of UI components and the two can work in tandem? If the former I feel like we should implement this slider with React Bootstrap to stay consistent, but if not then LGTM

Reply:
React Bootstrap doesn't have a slider library but it might be a good idea to migrate to Material-UI later down the line.

Reply:
Sounds good. Yeah, I've only worked with Material-UI before so I'm definitely open to discussing a switch / refactor